lib/agent0/agent0/logging/rollbar_utilities.py

```diff
@@ -0,0 +1,34 @@
+"""Utilities for rollbar."""
```

Comment:
Should this live in `hyperlogs`? that's where all the other logging stuff lives. It does seem to be agent0 specific, although if any other libs might want to use it then you should put it in hyperlogs.